### DISCUSSION TITLE
supabase-operator

### DISCUSSION THREAD

**Discussion by DTvGe45vc** [supabase/supabase#3908] [General]
Hello
I want to run some supabases in my kubernetes cluster.
Do you have a supabase operatory? Where I could see it?

Thank you.

**Comment by alexander-potemkin** [comment]
Ability to run Supabase in Kubernetes with operator would be very beneficial, indeed. Any chance Supabase team can share the development or at least an architecture here, please?

**Comment by Gaspero** [comment]
I was also looking for Kubernetes operator or at least Helm chart. Found following PRs/Discussions:
- #4881 
- https://github.com/supabase-community/supabase-kubernetes/pull/2
